src/ontology/mondo-edit.obo

```diff
@@ -20078,34 +20078,25 @@ is_a: MONDO:0001370 {source="DOID:115"} ! pericardial effusion
 [Term]
 id: MONDO:0001298
 name: congenital mitral valve insufficiency
-def: "Dysfunction of the mitral valve characterized by incomplete valve closure." [NCIT:P378]
+def: "Mitral valve insufficiency that is present at birth." [NCIT:C197881]
 synonym: "congenital insufficiency of mitral valve" EXACT []
-synonym: "congenital mitral insufficiency" EXACT [DOID:11502]
+synonym: "congenital mitral insufficiency" EXACT [DOID:11502, NCIT:C197881]
```

Comment:
you should remove the DOID from this synonym since this x-ref is not equivalent to this Mondo term anymore.

Reply:
done